frontend/src/pages/forms/Form.jsx

```diff
@@ -5,8 +5,11 @@ import MultiplierSelector from 'components/MultiplierSelector';
 import { connectWallet } from 'services/wallet';
 import { handleTransaction } from 'services/transaction';
 import Spinner from 'components/spinner/Spinner';
+<<<<<<< HEAD
+=======
 // import StarMaker from 'components/StarMaker';
```

Comment:
please resolve conflicts here and everywhere, and test everything before your next commit

Reply:
I just realized that another contributor worked on the Form page for the desktop screen.
So that's what is causing the conflict because I worked on both mobile and desktop screen.
Should I use conditionals to render the mobile version in another component or implement responsiveness again for what the previous contributor worked on?

Reply:
if you worked on both mobile and desktop versions, just go through everything making sure it is working well, it is ok to keep your changes and override previous ones 

@wheval

Reply:
@whateverfw